docs/source/app_developers_guide/intro_xo_transaction_family.rst

```diff
@@ -0,0 +1,127 @@
+there is essentially no business application for *XO* per se, this example transaction
+family demonstrates the functionality of Sawtooth, and the code that implements it also 
+serves as a reference for building other transaction processors.
+
+How to play XO
+==============
+
+Game rules
+----------
+
+*Tic-tac-toe* is a two-player game in which players take alternating turns to mark spaces
+on a 3x3 grid. Conventionally, *player one* marks their spaces with an 'X' and *player two* marks their spaces with an 'O'; and *player one* always takes their turn first.
+
+A player wins the game by marking 3 spaces in a horizontal, vertical, or diagonal row.
+In the condition that all 9 spaces on the grid have been marked but no player has 
+achieved a winning board position, the game is considered a draw.
+
+Playing XO using the command-line interface
+-------------------------------------------
+
+In order to play *XO*, users issue commands on the *XO* command-line interface. These 
+commands are then sent through the CLI to the *XO* client for further processing.
+Instructions on how to use the CLI are detailed below:
+
+1. After starting up a validator, *XO* transaction processor, and the REST API (see `Installing and Running Sawtooth <http://intelledger.github.io/0.8/cli/xo.html>`_), 
+   each player must initialize *XO* locally using the ``xo init`` command. Initialization
+   information is stored locally in a file. This command has two options: ``--username``
+   and ``--url``.
+
+  - The ``--username`` option is the name of the player. If ``--username`` is not
+    specified, OS environment variables will be parsed to find a suitable value.
+  - The ``--url`` option should be set to the URL of the REST API. The *XO* client 
+    will send requests to this URL in order to determine the validity of each move and 
+    update the game state on the blockchain. If no URL is specified, the URL will 
+    default to ``http://127.0.0.1:8080``.
+  
+    .. note::
+
+      A user can use the ``xo reset`` command to delete local *XO* data. This includes 
+      the saved URL and username.
+
+2. A user creates a new game using the ``xo create [name]`` command.
+  
+  - The *name* argument is the identifier for the new game to be created.
+
+3. Players alternate turns using the ``xo take [name] [space]`` command to mark spaces on
+   the grid.
+
+  - The *name* argument is the identifier for an existing game.
+  - The *space* argument is the space that is to be marked, which is an integer in the
+    range of 1 through 9.
+
+    .. note::
+
+      The first player to issue an ``xo take`` command to a newly created game is
+      recorded by their username as *player one*. The second player to issue a ``take`` 
+      command is recorded by their username as *player two*.
+
+      Each time a user attempts to take a space, the transaction processor will verify
+      that their username matches the name of the player whose turn it is. This ensures 
```

Comment:
"the identity of the player attempting to take a space matches the identity of the player whose turn it is"